components/email/src/Email.svelte

```diff
@@ -845,30 +843,14 @@
     );
   }
 
-  let messagesContainerNode: HTMLElement | null = null;
   let latestMessageNode: HTMLElement | null = null;
+  $: latestMessageNode = messageRefs[activeThread?.messages?.length - 1];
```

Comment:
@krisgardiner I'd love your feedback on this change. I'm not sure if you tried this and it didn't work as expected?